### ISSUE TITLE
Download path variable doesn't work

### ISSUE THREAD

**Issue by projektile** [norkunas/youtube-dl-php#16]
I've tried to trace why, but I figured I'd just ask before I break my back over it. 
i've set my path as per instruction:
$dl->setDownloadPath('/home/user/downloads');
but every time the file ends up in my server directory and i can't seem to make it go where I want, without moving it in my own code which defeats the purpose of having it in yours and the option -o in youtube-dl.

**Comment by norkunas** [comment]
Hello @projektile, i've just tested and everything works fine for me.

test.php:
```php
<?php

$yt = new \YoutubeDl\YoutubeDl();
$yt->setBinPath('C:/bin/youtube-dl');
$yt->setDownloadPath('D:/');

$video = $yt->download('https://www.youtube.com/watch?v=C1Tcfn2ZEgg');
var_dump($video->getFile());
```

Output:
```
php -f test.php

object(SplFileInfo)#6 (2) {
  ["pathName":"SplFileInfo":private]=>
  string(65) "D:/Eric Saade - Masquerade [Official Music Video]-C1Tcfn2ZEgg.mp4"
  ["fileName":"SplFileInfo":private]=>
  string(62) "Eric Saade - Masquerade [Official Music Video]-C1Tcfn2ZEgg.mp4"
}
```

Could you provide your code how you use it?

**Comment by norkunas** [comment]
Maybe your process working dir somehow changes when downloading :question: and if that's the case I can introduce another option and transfer the file manually with php.

**Comment by projektile** [comment]
hey norkunas,

this is a cut from my index.php which invokes your api:
<code>
	# This is the function you will need to modify to include the youtube-dl API
	if(isset($_POST['name'])) {
		$object = $_POST['name'];

		$dl = new YoutubeDl([
			'continue' => true, // force resume of partially downloaded files. By default, youtube-dl will resume downloads if possible.
			'format' => 'bestaudio',
			'extract-audio' => true,
		]);

		$dl->setDownloadPath('/home/projektile/Music');

		try {
			$video = $dl->download('https://www.youtube.com/watch?v='.$object);
			echo $video->getTitle(); // Will return Phonebloks
		// $dl->getFile(); // \SplFileInfo instance of downloaded file
		} catch (NotFoundException $e) {
		// Video not found
		} catch (PrivateVideoException $e) {
		// Video is private
		} catch (CopyrightException $e) {
		// The YouTube account associated with this video has been terminated due to multiple third-party notifications of copyright infringement
		} catch (\Exception $e) {
		// Failed to download
		}
</code>

**Comment by norkunas** [comment]
@projektile thanks, can you print the output of $dl->getFile() ?

**Comment by projektile** [comment]
Sorry for my late response, It's been a busy last few days.
the only way ive been able to print the output of $dl->getFile() gives me:
<code>
SplFileInfo Object
(
    [pathName:SplFileInfo:private] => /home/projektile/Music
    [fileName:SplFileInfo:private] => Music
)
</code>
i can redirect it to /tmp/ instead of my music directory and it will work, which seems odd because this doesn't seem to be a permission issue either.

**Comment by norkunas** [comment]
@projektile so when you got this output ``` SplFileInfo Object ( [pathName:SplFileInfo:private] => /home/projektile/Music [fileName:SplFileInfo:private] => Music )``` the file was downloaded to the primary directory where the script was executed?

**Comment by projektile** [comment]
Yes, the file was saved to /usr/share/nginx/html/ with that spl info. the only two directories I've managed to save the file into is either /usr/share/nginx/html/, or /tmp/. for my purposes /tmp/ will work but I'm still curious why I'm having this problem.
my environment is:
Linux 4.1.15
nginx 1.8.0
PHP 7.0.2

**Comment by norkunas** [comment]
@projektile i made a temporary ugly update, could you try now to pull ```dev-master``` version and set ```$dl->setMoveWithPhp(true);``` and check if it works?
